pint.json

```diff
@@ -1,12 +1,8 @@
 {
     "preset": "laravel",
     "rules": {
-        "class_attributes_separation": false,
         "concat_space": false,
         "not_operator_with_successor_space": false,
-        "nullable_type_declaration_for_default_null_value": false,
-        "ordered_imports": false,
-        "phpdoc_align": false,
-        "phpdoc_separation": false
+        "ordered_imports": false
```

Comment:
Why is `ordered_imports` still disabled?

Reply:
Separate PR